azkaban-common/src/main/java/azkaban/jobExecutor/AbstractProcessJob.java

```diff
@@ -81,6 +84,31 @@ protected void resolveProps() {
     jobProps = PropsUtils.resolveProps(jobProps);
   }
 
+  /**
+   * prints the current Job props to the Job log.
+   */
+  protected void logJobProperties() {
```

Comment:
thought it as well at the first place, we now depend on a job prop value to determine if the content should be print or not and for other props we may want to have different indicators to make this judge, so I would recommend ship the code as for now and  it is fairly an easy change if we have request to support more in a latter time. plz let me know if  this makes sense.